Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -25,7 +25,7 @@ enum ReaderModeState: String {
     case active = "Active"
 }
 
-enum ReaderModeTheme: String {
+enum ReaderModeTheme: String, Equatable {
```

Comment:
Why do we need to make this equatable? If it's only for the tests then we can make this equatable only in the tests target. Same comment for `ReaderModeFontType`, `ReaderModeFontSize` and `ReaderModeStyle` protocol comformance.

Reply:
Update: `Extension outside of file declaring struct '...' prevents automatic synthesis of '==' for protocol 'Equatable'`. Doing it outside the file means the implementation will not be automatically synthesized. Are we sure we want to do this?